.gitignore

```diff
@@ -38,3 +38,4 @@ dist
 
 # Optional REPL history
 .node_repl_history
+package-lock.json
```

Comment:
should we consider committing this?

Reply:
It is not necessary but in the end it is a file that we should not upload, I saw it convenient to add it to git ignore, but i can remove this from the file.